VotingApplication/VotingApplication.Web.Api/Scripts/Poll.js

```diff
@@ -233,14 +193,14 @@
                     Name: username,
                     PollId: pollId,
                     Token: {
-                        TokenGuid: tokenId
+                        TokenGuid: token || Common.sessionItem("token", pollId)
```

Comment:
The top of the file already does this, so we should just need "TokenGuid: token"